File: Day3/Code/Palindrome.cs
public class Solution {
    public bool IsPalindrome(int x) {
        if(x<0)
            return false;
        else{
            int temp = x;
            int reverse = 0;
            while(temp!=0){
                int rem = temp%10;
                reverse = reverse*10 + rem;
                temp /= 10;
            }
            if(reverse==x){
                return true;
            }
            else
                return false;
        }
    }
}

File: Day01/Palindrome.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0) return false;
        else{
            String val = String.valueOf(x);
            int n = val.length();
            for(int i=0;i<n/2;i++){
                if(val.charAt(i)==val.charAt(n-i-1)){
                    continue;
                }
                else{
                    return false;
                }
            }
            return true;
        }
    } 
}